packages/runtime-core/src/componentEmits.ts

```diff
@@ -226,7 +226,9 @@ export function normalizeEmitsOptions(
   }
 
   if (!raw && !hasExtends) {
-    cache.set(comp, null)
+    if (comp && typeof comp === 'object') {
```

Comment:
I think you can use the `isObject()` function from `@vue/shared`

Reply:
Thanks ❤️